### ISSUE TITLE
Cannot use ConfigModule as described in docs

### ISSUE THREAD

**Issue by jmcdo29** [nestjs/config#82]
## I'm submitting a...

<pre><code>
[ ] Regression 
[X] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

[According to the docs](https://docs.nestjs.com/techniques/configuration#using-the-configservice) after importing `ConfigModule.forRoot()` in my `AppModule`, I should be able to import `ConifgModule` in any `FeatureModule` and in `FeatureService` be able to inject `ConfigService`. Currently, the following error appears:
```sh
[Nest] 20030   - 02/17/2020, 4:28:04 PM   [NestFactory] Starting Nest application...
[Nest] 20030   - 02/17/2020, 4:28:04 PM   [InstanceLoader] ConfigModule dependencies initialized +24ms
[Nest] 20030   - 02/17/2020, 4:28:04 PM   [ExceptionHandler] Nest can't resolve dependencies of the BasicService (?). Please make sure that the argument ConfigService at index [0] is available in the BasicModule context.

Potential solutions:
- If ConfigService is a provider, is it part of the current BasicModule?
- If ConfigService is exported from a separate @Module, is that module imported within BasicModule?
  @Module({
    imports: [ /* the Module containing ConfigService */ ]
  })
 +1ms
Error: Nest can't resolve dependencies of the BasicService (?). Please make sure that the argument ConfigService at index [0] is available in the BasicModule context.

Potential solutions:
- If ConfigService is a provider, is it part of the current BasicModule?
- If ConfigService is exported from a separate @Module, is that module imported within BasicModule?
  @Module({
    imports: [ /* the Module containing ConfigService */ ]
  })

    at Injector.lookupComponentInParentModules (~/nest-config-test/node_modules/@nestjs/core/injector/injector.js:190:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Injector.resolveComponentInstance (~/nest-config-test/node_modules/@nestjs/core/injector/injector.js:146:33)
    at async resolveParam (~/nest-config-test/node_modules/@nestjs/core/injector/injector.js:100:38)
    at async Promise.all (index 0)
    at async Injector.resolveConstructorParams (~/nest-config-test/node_modules/@nestjs/core/injector/injector.js:115:27)
    at async Injector.loadInstance (~/nest-config-test/node_modules/@nestjs/core/injector/injector.js:79:9)
    at async Injector.loadProvider (~/nest-config-test/node_modules/@nestjs/core/injector/injector.js:36:9)
    at async Promise.all (index 3)
    at async InstanceLoader.createInstancesOfProviders (~/nest-config-test/node_modules/@nestjs/core/injector/instance-loader.js:41:9)
```

## Expected behavior

As following the docs, it would be possible to just import `ConfigModule.forRoot()` and then use `imports: [ConfigModule]` in any feature module to have access to the `ConfigService`.

## Minimal reproduction of the problem with instructions

Reproduction: https://github.com/jmcdo29/nest-config-test
Reproduction steps: [In the readme](https://github.com/jmcdo29/nest-config-test#nest-config-test) but here as well:
1) `git clone https://github.com/jmcdo29/nest-config-test.git`
2) `npm i`
3) `npm run start:dev` or `npm build && npm start`
4) observe error 

## What is the motivation / use case for changing the behavior?

Mainly, I just see a lot of questions either on Discord or SO about using the `ConfigModule` as the docs show and the answer is always add `isGlobal: true` to make it work. While that works, it seems this kind of goes against the spirit of being explicit about what is being imported where. 

## Environment

<pre><code>
Nest versions: 
Common: 6.11.7
Core: 6.11.7
Platform-Express: 6.11.7
Config: 0.2.2

 
For Tooling issues:
- Node version: 13.7.0
- Platform: Linux Mint 19.1

</pre></code>

Other:
I know the workarounds for this are to set `isGlobal: true` in configuration, or to use a `CommonModule` format where `ConfigModule.forRoot()` is `import`ed and `ConfigModule` is `export`ed, which is fine if that is the final solution, but it should be documented that that's the correct way to manage the `ConfigModule`. E.g.

```ts
@Module({
  imports: [ConifgModule.forRoot()],
  exports: [ConfigModule]
})
```

**Comment by kamilmysliwiec** [comment]
Thanks for reporting @jmcdo29 :) Fixed in 0.2.3

**Comment by jmcdo29** [comment]
Confirmed! Thanks for the quick turnaround!
